<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderEncryptDecryptDialog</class>
 <widget class="QDialog" name="FolderEncryptDecryptDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encrypt Folder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/resources/images/DCCrypt_Logo.png</normaloff>:/images/resources/images/DCCrypt_Logo.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="toolTip">
        <string>The folder that will be encrypted/decrypted.
But notice, that only the content will be encrypted/decrypted and not the folder it self.</string>
       </property>
       <property name="text">
        <string>Input Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputLineEdit">
       <property name="toolTip">
        <string>The folder that will be encrypted/decrypted.
But notice, that only the content will be encrypted/decrypted and not the folder it self.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseInputButton">
       <property name="toolTip">
        <string>The folder that will be encrypted/decrypted.
But notice, that only the content will be encrypted/decrypted and not the folder it self.</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="inputFolderStatusLabel">
       <property name="text">
        <string>No folder chosen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="toolTip">
        <string>The folder that will be the product of the operation.</string>
       </property>
       <property name="text">
        <string>Output Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputLineEdit">
       <property name="toolTip">
        <string>The folder that will be the product of the operation.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseOutputButton">
       <property name="toolTip">
        <string>The folder that will be the product of the operation.</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outputFolderStatusLabel">
       <property name="text">
        <string>No folder chosen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="keyLabel">
         <property name="toolTip">
          <string>The file that will be used to encrypt/decrypt (the content of) the &quot;Input Folder&quot;.</string>
         </property>
         <property name="text">
          <string>Key File:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="repeatPasswordLabel">
         <property name="toolTip">
          <string>The password that will be used to encrypt/decrypt (the content of) the &quot;Input Folder&quot;.</string>
         </property>
         <property name="text">
          <string>Repeat Password:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="keyLineEdit">
         <property name="toolTip">
          <string>The file that will be used to encrypt/decrypt (the content of) the &quot;Input Folder&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="repeatPasswordEdit">
         <property name="toolTip">
          <string>The password that will be used to encrypt/decrypt (the content of) the &quot;Input Folder&quot;.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="browseKeyButton">
       <property name="toolTip">
        <string>The file that will be used to encrypt/decrypt (the content of) the &quot;Input Folder&quot;.</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="keyFileStatusLabel">
       <property name="text">
        <string>No file chosen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="encryptWithLabel">
       <property name="toolTip">
        <string>Choose if you want to encrypt/decrypt the (content of the) folder with a &quot;Key File&quot; or a password.</string>
       </property>
       <property name="text">
        <string>Encrypt/Decrypt with:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="keyFileRadioButton">
       <property name="toolTip">
        <string>Encrypt/Decrypt with a &quot;Key File&quot;.</string>
       </property>
       <property name="text">
        <string>Key File</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="passwordRadioButton">
       <property name="toolTip">
        <string>Encrypt/Decrypt with a password.</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Encrypt/Decrypt</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progressGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Encrypting/Decrypting...</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="stopSlider">
        <property name="toolTip">
         <string>Slide to the right to cancel the operation.</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="shutdownCheckBox">
        <property name="text">
         <string>Shutdown computer after the operation is finished</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>Visit our website.</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://www.dccrypt.herobo.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.dccrypt.herobo.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>browseInputButton</tabstop>
  <tabstop>outputLineEdit</tabstop>
  <tabstop>browseOutputButton</tabstop>
  <tabstop>keyLineEdit</tabstop>
  <tabstop>repeatPasswordEdit</tabstop>
  <tabstop>browseKeyButton</tabstop>
  <tabstop>keyFileRadioButton</tabstop>
  <tabstop>passwordRadioButton</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>shutdownCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
